src/app/app.component.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Climate Change Lab
+ * App Component
+ */
+import { Component, ViewEncapsulation } from '@angular/core';
+import {HmrState} from 'angular2-hmr';
```

Comment:
Angular-hmr uses webpack hmr under the hood, so effectively it's a duplication unless we want the additional "side-effects" of angular-hmr which I am awaiting further clarification on from some Angular people